### ISSUE TITLE
STORY: Configure Author/Creator Facet

### ISSUE THREAD

**Issue by lovinscari** [emory-libraries/blacklight-catalog#197]
**Status**: facet exists already; field mapping and subfields need refinement

As an end user, I would like to search or browse materials by their creators or other notable contributors, so that I can target my search to match specific people or organizations responsible for creating the material

**Acceptance criteria**

- [x] Facet heading should be “Author/Creator”
- [x] The facet search should find main authors/creators as well as secondary authors/creators
- [x] Facet should follow the general functionality as noted in [Facet - general wireframe](https://www.figma.com/file/ocSo3KytI6GgwjalSVAmE1/Blacklight-Wireframes?node-id=91%3A3920) and [Facet - Modal window wireframes](https://www.figma.com/file/ocSo3KytI6GgwjalSVAmE1/Blacklight-Wireframes?node-id=91%3A3765)
- [x] Displays on search result pages only (not on homepage)

**SOLR/Metadata Notes:**

- [x] See placeholder field on [worksheet # 94](https://docs.google.com/spreadsheets/d/1jSqgNd6HnVJdJMrrShAofkmK8uGgHGdlKODmZOL5NfY/edit#gid=1979033236&range=42:42). This would include both primary and secondary creator names (1XX, 7XX) beyond what is contained in author_display_ssim
- [x] Values should not be tokenized

**Comment by devanshu-m** [comment]
PR: https://github.com/emory-libraries/blacklight-catalog/pull/213

**Comment by lovinscari** [comment]
@devanshu - @eporter23 is still testing this but is finding a few abnormalities which she is not sure at this point where the issue is occurring. She will likely provide more details during standup today.

**Comment by eporter23** [comment]
@devanshu-m @lovinscari I wrote up some of what I saw yesterday while testing this. I think the facet is configured as specified. The anomalies I am seeing probably boils down to 1) the data as entered and 2) Blacklight's sort behavior. Note: I tested for over an hour and didn't have time to continue further.  I would suggest we write a new ticket to investigate some of these behaviors further to find out what options we have to make this facet more usable to end users. [Test notes](https://docs.google.com/document/d/1KLqfzB0Oa8C_U4-r90uMgt182k6ruT9-niVl-LjXQ7s/edit?usp=sharing)
